R/keras.R

```diff
@@ -0,0 +1,42 @@
+#' @rdname vetiver_create_description
+#' @export
+vetiver_create_description.keras.engine.training.Model <- function(model) {
+    model_config <- model$get_config()
+    glue("A {model_config$name} keras model with {length(model_config$layers)} layers")
+}
+
+#' @rdname vetiver_create_meta
+#' @export
+vetiver_create_meta.keras.engine.training.Model <- function(model, metadata) {
+    vetiver_meta(metadata, required_pkgs = "keras")
+}
+
+#' @rdname vetiver_create_ptype
+#' @export
+vetiver_ptype.keras.engine.training.Model <- function(model, ...) {
+    NULL
```

Comment:
@t-kalinowski one question I have is whether it would be better to have a default of `NULL` for the input data prototype (schema) which is better for non-rectangular data like images or whether we should have a default of rectangular data and get some data from the model or user. Either way, users can opt out of the default via the `save_prototype` argument, so this is a question about defaults.